vue/src/modules/setup/invitations/UserInvitations.vue

```diff
@@ -32,19 +32,8 @@
             }">{{ data.value }}</span>
             </template>
             <template v-slot:cell(actions)="data">
-              <b-dropdown variant="transparent text-dark p-0" right no-caret>
-                <template slot="button-content">
-                  <i class="fas fa-ellipsis-v"></i>
-                </template>
-                <b-dropdown-item @click="editUser(data.item)" v-if="data.item.status !== PENDING_STATUS">
-                  <i class="fas fa-pencil-alt"></i>
-                  {{ $t('Edit') }}
-                </b-dropdown-item>
-                <b-dropdown-item @click="deleteUser(data.item)">
-                  <i class="far fa-trash-alt"></i>
-                  {{ $t('Delete') }}
-                </b-dropdown-item>
-              </b-dropdown>
+              <i class="fas fa-pencil-alt mx-3 mx-sm-0 mx-md-3 mx-lg-0 text-primary hover-pointer" @click="editUser(data.item)" v-if="data.item.status !== PENDING_STATUS"/>
```

Comment:
I think we don't need `mx-3 mx-sm-0 mx-md-3 mx-lg-0`. Why do we need that? We only need 'mr-3`